src/mono/wasm/templates/templates/browser/wwwroot/main.js

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+import { dotnet } from './_framework/dotnet.js'
+
+const { setModuleImports, getAssemblyExports, getConfig } = await dotnet
+    .withStartupOptions({})
```

Comment:
I'm tackling with this my original PR. It shouldn't be in the template

Reply:
I can remove this from this draft PR when your changes land.